mina-payout-reports/templates/deployment-api.yaml

```diff
@@ -0,0 +1,143 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "mina-payout-reports.fullname" . }}-api
+  labels:
+    {{- include "mina-payout-reports.labels" . | nindent 4 }}-api
+spec:
+  {{- with .Values.payoutReportsApi }}
+  replicas: {{ .replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "mina-payout-reports.selectorLabels" $ | nindent 6 }}-api
+  template:
+    metadata:
+      {{- with .podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "mina-payout-reports.selectorLabels" $ | nindent 8 }}-api
+    spec:
+      {{- with .imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "mina-payout-reports.serviceAccountName" $ }}
+      securityContext:
+        {{- toYaml .podSecurityContext | nindent 8 }}
+      initContainers:
+        - name: initdb
+          image: "{{ .image.repository }}:{{ .image.tag }}"
```

Comment:
**suggestion:** At rendering, this tag is empty, which is leading to invalid image format.

```
        image: '673156464838.dkr.ecr.us-west-2.amazonaws.com/mina-payout-reports:'
```